### PR TITLE
Outsource init code and add tests for IntervalArithmetic; unsupport :fast multiplication

### PR DESCRIPTION
**Pull request by schillic**
~The first commit is from #233.~

### CODE REVIEW

File: src/init_IntervalArithmetic.jl [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@reexport using IntervalArithmetic
```
Comment (mforets) [inline src/init_IntervalArithmetic.jl:1]: This is quite complex, i wish we can keep the latest and greatest version :)

**Review by mforets — APPROVED** [review]
is "fast matrix IA power" not a thing anymore upstream?

**Comment by schillic** [comment]
> is "fast matrix IA power" not a thing anymore upstream?

I cannot say, but they removed the `:fast` feature.

File: src/init_IntervalArithmetic.jl (hunk 1 above)
Comment (schillic) [reply src/init_IntervalArithmetic.jl:1]: I only moved the code here (exactly because I thought it was quite complex).

**Pull request merged**